src/screen/ConnectionsList/Connection/styles.ts

```diff
@@ -52,6 +56,53 @@ export const Container = styled.li<IContainerProps>`
       `}
   }
 
+  .connection-actions-menu {
+    width: 220;
```

Comment:
I think we should keep the pattern and use `220px` instead of `220`

Reply:
Oops